requirements.txt

```diff
@@ -1,5 +1,7 @@
 # Basic Python requirements for Garden Linux
 # that can be installed by Python pip.
 # Note: You should always prefer your local package manager.
-PyYAML
+json
```

Comment:
not a pip package, it should be [built-in](https://docs.python.org/3/library/json.html)

installing with `pip install -r requirements` would lead to 
```
ERROR: Could not find a version that satisfies the requirement json (from versions: none)
```

we can just remove `json` from the requirements.txt (same for yaml)

Reply:
Removed json and yaml requirements.